specification/cognitiveservices/data-plane/FormRecognizer/readme.md

```diff
@@ -18,7 +18,9 @@ openapi-type: data-plane
 These settings apply only when `--tag=release_1_0` is specified on the command line.
 
 ``` yaml $(tag) == 'release_1_0'
-input-file: preview/v1.0/FormRecognizer.json
+input-file: 
```

Comment:
Though it's implied, it is a good idea to include ocr.json

Reply:
Previously, we were hoping to only reference a subset of definitions from the CV ocr.json.  But as the build validation scripts end up validating even unreferenced definitions, we are removing them.  So we can add ocr.json explicitly to the list of input files.

Reply:
A. Why does ocr.json have unreferenced definitions?
B. Regardless of whether these definitions are used, violations must be addressed

Reply:
@ramparab could you address this too?

Reply:
done.